src/components/stepper/step-label/step-label.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { expect, fixture } from '@open-wc/testing';
```

Comment:
- Add inner text in fixture
- Use equalSnapshot
- Add a case with icon and one with disabled
- Add the a11y test